### PR TITLE
DOCSP-27982 Import and Export Compass updates

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Update Import and export procedures to reflect new updates. 

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27982/import-export/

## JIRA
https://jira.mongodb.org/browse/DOCSP-27982

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64650ab7df0f8173e97e3523

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/import-export.txt [hunk 1]
```diff
@@ -189,17 +194,38 @@ Procedure
 
          .. step:: Click :guilabel:`Export`.
 
-         .. step:: Choose the appropriate file type.
+         .. step:: Select the appropriate file type.
+
+            Under :guilabel:`Export File Type`, select either :guilabel:`JSON` 
+            or :guilabel:`CSV`. If you select :guilabel:`JSON`, you can expand 
+            the :guilabel:`Advanced JSON Format` dropdown and select from the 
+            following extended JSON formats`:`
```
Comment (jason-price-mongodb) [inline source/import-export.txt:202]: Omit the quotes from `:`?

File: source/includes/steps-export-filter.yaml [hunk 2]
```diff
@@ -14,46 +14,38 @@ content: |
...
+  entered in the query bar. 
 ---
 title: Select document fields to include in your exported file.
 level: 4
 ref: select-export-fields
 content: |
+
+  You can choose to export :guilabel:`All Fields` or :guilabel:`Select fields in table`.
+  In the query bar, you can also use the :guilabel:`Project` field to specify 
+  which fields to return or export.
```
Comment (jason-price-mongodb) [inline source/includes/steps-export-filter.yaml:39]: "the fields"

File: source/includes/steps-export-filter.yaml [hunk 3]
```diff
@@ -64,24 +56,45 @@ content: |
...
 level: 4
 ref: click-file-type
 content: |
-  Under :guilabel:`Select Export File Type`, select either 
-  :guilabel:`JSON` or :guilabel:`CSV`. If you select :guilabel:`JSON`,
-  your data is exported to the target file as a comma-separated 
-  array of JSON objects.
+  Under :guilabel:`Export File Type`, select either :guilabel:`JSON` or :guilabel:`CSV`.
+  If you select :guilabel:`JSON`, you can expand the :guilabel:`Advanced JSON Format` dropdown 
+  and select from the following extended JSON formats`:`
```
Comment (jason-price-mongodb) [inline source/includes/steps-export-filter.yaml:65]: See earlier comment.

File: source/import-export.txt [hunk 4]
```diff
@@ -117,10 +117,13 @@ To import your formatted data into a collection:
 
 A progress bar displays the status of the import. If an error occurs
 during import, the progress bar turns red and an error message
-appears in the dialog. After successful import, the dialog closes and
+appears in the dialog. You can click :guilabel:`View Log` to see all errors.
```
Comment (jason-price-mongodb) [inline source/import-export.txt:120]: "To see XXX, click YYY."

**Review by jason-price-mongodb — APPROVED** [review]
@jocelyn-mendez1  LGTM with comments. Cheers.

File: source/import-export.txt [hunk 5]
```diff
@@ -134,11 +137,13 @@ pipeline results.
...
 
-While it is possible to exclude documents by using a query filter,
-it is not possible to re-shape exported documents with a
-:ref:`project <query-bar-project>` document. Even when you specify a
-``project`` option in the query, |compass-short| still exports the
-entire document.
+- Avoid exporting to :guilabel:`CSV` files when possible. :guilabel:`CSV` files 
+  may lose type information and are not suitable for backing up your data.
+
+- You can use a query filter to export only the documents which match the filter.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/import-export.txt]: Done.

File: source/import-export.txt [hunk 6]
```diff
@@ -134,11 +137,13 @@ pipeline results.
...
+
+- You can use the :ref:`Project <query-bar-project>` field in the query bar to 
+  specify which fields to return or export.
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/import-export.txt]: Done.

File: source/import-export.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/import-export.txt]: Done.

File: source/includes/steps-export-filter.yaml (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/includes/steps-export-filter.yaml]: Done.

File: source/includes/steps-export-filter.yaml (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/includes/steps-export-filter.yaml]: Done.

File: source/import-export.txt (hunk 4 above)
Comment (jocelyn-mendez1) [reply source/import-export.txt]: Done.

**Review by lerouxb — APPROVED** [review]

**Pull request merged**